coverage/clover.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should ignore the coverage folder. 

There is no need for tracking this folder.